apps/web/package.json

```diff
@@ -52,6 +52,7 @@
     "raf": "^3.4.1",
     "react-native-qrcode-svg": "^6.2.0",
     "react-native-reanimated": "*",
+    "react-native-safe-area-context": "4.7.4",
```

Comment:
can we move this dependency to root `node_modules` and use in `peerDependencies` as it's used by both in packages and apps?